internal/runner/job.go

```diff
@@ -37,13 +39,29 @@ func (rnr *Runner) runJob(j *Job, wg *sync.WaitGroup) {
 	L := rnr.createLuaState(j)
 	defer L.Close()
 
+	withTimeout := j.script.Timeout != 0
+	var (
+		ctx    context.Context
+		cancel context.CancelFunc
+	)
+
 	for {
+		if withTimeout {
+			ctx, cancel = context.WithTimeout(context.Background(), j.script.Timeout)
+			defer cancel()
```

Comment:
We already have 'cancel()' below on #62. Also, this defer uses in (maybe) long loop.

Reply:
This doesn't hurt and should be done anyway. Tight defer shouldn't be problematic anymore since Go 1.14 https://golang.org/doc/go1.14 or am I wrong?

Reply:
Anyway this line is duplicated. I think, we can remove it

Reply:
Not sure, if a j.stop signal is received we return and it could be possible that we leak a context.

Reply:
a stop signal we process after line #62

Reply:
After create a context we not have return statement

Reply:
Yeah true. Ok let me remove it.